app/app/crashes/[record_locator]/page.tsx

```diff
@@ -59,6 +60,12 @@ export default function CrashDetailsPage({
     <>
       <AppBreadCrumb />
       <CrashHeader crash={crash} />
+      {
+        // show alert if crash on private drive or outside of Austin full purpose
+        (crash.private_dr_fl || !crash.in_austin_full_purpose) && (
```

Comment:
`|| !crash.in_austin_full_purpose` → "or not in austin full purpose"

That field renaming makes your code just jump off the page in legibility.